.template/addons/bootstrap/stylesheets/vendor/_bootstrap.scss

```diff
@@ -5,7 +5,10 @@
 
 // Configuration
 @import 'bootstrap/scss/functions';
+// @import 'variables'; // Optional variable overrides here
```

Comment:
Now, this file is imported in the manifest file, so I am curious about how the `application.scss` would look like:

![image](https://user-images.githubusercontent.com/696529/219248382-2fd2cbc9-512a-4a39-86d9-c454ceab2e89.png)

https://github.com/onward-venture/ewa-payroll-web/blob/c520ab422a4a24c6a2fb31d6aca50bf8ec27c548/app/assets/stylesheets/_core.scss#L2-L8

Reply:
Nice catch, I missed this one!

Reply:
Let me remove this line at the moment, as at the moment we always use `variables` without the `bootstrap/scss/functions` (the reason why we can't use `rem()` function in the variables. 

I will open a new issue for this case, as I believe it is a nice improvement but I need to research more about how to have a clean import (I'm thinking having 2 vendors import in application.scss in order to keep the `variables` import outside from the `vendro` folder 👍 )

Reply:
3dca276

Reply:
... and issue created in https://github.com/nimblehq/rails-templates/issues/393